hibernate-core/src/main/java/org/hibernate/mapping/Constraint.java

```diff
@@ -151,12 +151,8 @@ public int getColumnSpan() {
 	public Column getColumn(int i) {
 		return  columns.get( i );
 	}
-	//todo duplicated method, remove one
-	public Iterator<Column> getColumnIterator() {
-		return columns.iterator();
-	}
```

Comment:
As in another PR you created, we need to consider backward compatibility. Again, it might be a good idea to merge into wip/v6.0.
The above deletion might break our client's code, IMHO.

Reply:
Yes, this was done as idea to include it in wip/v6.0.

Reply:
Then you might need to change the merging destination accordingly.

Reply:
My bad, that has been done. Thank you for pointing it out.